src/core/chat-api.ts

```diff
@@ -31,6 +38,19 @@ interface SendMessageParams {
   headers?: MessageHeaders;
 }
 
+interface DeleteMessageParams {
+  hard?: boolean;
+  description?: string;
+  metadata?: MessageDetailsMetadata;
+}
+
+export interface DeleteMessageResponse {
+  /**
+   * The timestamp of when the deletion occurred.
+   */
+  deletedAt: number;
```

Comment:
Perhaps a realtime question too - should this return the client id of who managed to successfully delete too? What happens if two people delete at the same time?

Reply:
I've added the clientId to the struct, will make the realtime change now.